lib/manageiq/providers/openstack/legacy/openstack_handle/handled_list.rb

```diff
@@ -15,7 +16,12 @@ def handled_list(collection_type, options = {})
       # @orchestration_service.handled_list(:resources, :stack => stack)
       # By default, it calls :all method on the Fog collection, that has unified interface in all list methods in Fog
       # and always returns detailed list.
-      multi_tenancy_class.new(self, @os_handle, self.class::SERVICE_NAME, collection_type, options,
+      multi_tenancy_class_ = if all_tenants
+                               OpenstackHandle::MultiTenancy::Option
+                             else
+                               multi_tenancy_class
+                             end
```

Comment:
The extra _ in the variable seems a bit weird to me - maybe call it 'multi_tenancy_class' and rename the function to 'default_multi_tenancy_class' or something?